browser/src/control/Toolbar.js

```diff
@@ -584,6 +584,21 @@ L.Map.include({
 				this.filterResults(searchTerm, isAnyMatchingContent, id);
 			}
 		}.bind(this));
+
+
+		const buttons = document.querySelectorAll('.scroll-button');
+
+		buttons.forEach((button) => {
+			button.addEventListener('click', () => {
+				const targetId = button.getAttribute('data-target');
```

Comment:
"button.dataset.target" should also work :) That is the strong side of these attributes i guess.